fe/java-udf/src/main/java/org/apache/doris/udf/UdfUtils.java

```diff
@@ -214,7 +213,7 @@ public static void copyMemory(
     }
 
     public static URLClassLoader getClassLoader(String jarPath, ClassLoader parent) throws MalformedURLException {
-        URL url = new File(jarPath).toURI().toURL();
+        URL url = new URL(jarPath);
```

Comment:
Add a rule to check whether this jar exists

Reply:
done